index.html

```diff
@@ -126,7 +126,58 @@
+                                        </ol>
+                                <p>
+                                        Nadużywanie metod magicznych:
+                                </p>
+                                        <ol>
+                                                <li>Wspomniane wyżej połączenie dwóch query builderów.</li>
+                                                <li><a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Auth/AuthManager.php">AuthManager</a> również pozwala nam wykonywać <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Support/Manager.php#L137">zgoła nieoczekiwane operacje</a> na <a href="http://laravel.com/docs/5.0/authentication#retrieving-the-authenticated-user">obiektach</a>, o których nie mamy większego pojęcia. <a href="">Dyskusja</a> (dotyczyła wersji 4.2, ale wygląda na to, że nadal jest aktualna)</li>
+                                                <li>Mockowanie tych szalonych zależności wymaga intensywnego korzystania z xdebuga.</li>
+
+                                        </ol>
+                                <p>
+                                        Występują też w przyrodzie zjawiska, których po prostu nie rozumiem:
+                                </p>
+                                        <ol>
+                                                <li>Packagist laravelish enough is not! W związku z czym istenieje <a href="http://packalyst.com/">Packalyst</a></li>
+                                        </ol>
+                                <p>
+                                Dlaczego w takim razie ktoś w ogóle używa Laravela? Z bardzo prostej przyczyny - Otwell zapewnia projektowi fenomenalny wręcz marketing, próg wejścia jest w zasadzie zerowy (najwyżej opakujesz kod proceduralny w statyczne metody), a do tego czas nas czasem goni... Co zrobić? Jak żyć? Nie używać Laravela do żadnych projektów, które potem będą musiały być przez dłuższy czas utrzymywane. Twoim celem jest zwiększenie sprzedaży piwa marki X w wakacje, wśród studentów? Zastanów się nad tym, czy nie byłoby warto użyć <a href="http://silex.sensiolabs.org/">Silexa</a> albo <a href="http://www.slimframework.com/">Slima</a>, a jeśli nie chcesz bądź po prostu musisz - to piszże w Laravelu.<br>
+                                Nigdy, przenigdy, nie używaj Laravela do projektów, które będą miały być utrzymywane przez wiele lat. Słaba architektura Cię wykończy zanim zdąży Cię wykończyć brak dobrych testów jednostkowych.
+                                </p>
+                                <p>
+                                        Na koniec niestety przypomniałem sobie, że <a href="https://github.com/laravel/framework/pull/4824">korzystanie z DI</a> też potrafi skrócić życie. Bardzo długo szukałem w kodzie PHPowym zmiennej $driver przekazywanej do konstruktora, a ona sobie spokojnie żyła <a href="https://devel-m6w6.rhcloud.com/mdref/http/Client/__construct">w C</a>.
```

Comment:
Tam jest dość konkretna lista ułomności. Forma ma jeden cel - ułatwić użytkownikowi zrozumienie mojego przekazu i uważam, że ją to usprawiedliwia.